src/com/company/lambdaexamples/LambdaExpressions3.java

```diff
@@ -0,0 +1,90 @@
+package com.company.lambdaexamples;
+
+import java.util.*;
+
+public class LambdaExpressions3 {
+    public static void main(String[] args) {
+        List<Integer> intNumbers = new ArrayList<>();
+        intNumbers.add(100);
+        intNumbers.add(-999);
+        intNumbers.add(-9);
+        intNumbers.add(8);
+        intNumbers.add(80);
+        taskOne(intNumbers);
+        Person firstPerson = new Person("Iliana Markova", 165);
+        Person secondPerson = new Person("Test Testov", 180);
+        Person thirdPerson = new Person("LoremLoremLorem  LoremLoremLorem", 155);
+        List<Person> listPerson = new ArrayList<>();
+        listPerson.add(firstPerson);
+        listPerson.add(secondPerson);
+        listPerson.add(thirdPerson);
+        addPersonNames(listPerson);
+        getLongNameCharacter(listPerson);
+        getHighestPerson(listPerson);
+        getSquareNumbers(intNumbers);
+    }
+
+
+//1. Направете метод, който за List от Integer-и, да намира дали има число, което е трицифрено.
+
+    public static void taskOne(List<Integer> listNumbers) {
+        listNumbers.forEach((n) -> {
+            if (n >= 100 && n <= 999 || n >= -999 && n <= -100) {
+                System.out.println(String.format("3 digit number is %s", n));
+            }
+        });
+
+    }
+//2. Нека имаме клас Person и всеки Person да има име. Направете метод, който по List да създава списък с имената на хората.
+
+    public static void addPersonNames(List<Person> personObjects) {
+        List<String> personNames = new ArrayList<>();
```

Comment:
Another option is to use stream().map() and collect() functions:
`List<String> personNames = personObjects.stream()
				                                        .map(p -> p.getName())
				                                        .collect(Collectors.toList());`

Reply:
code block is changed as it is suggested